### ISSUE TITLE
Prime World (235340)

### ISSUE THREAD

**Issue by vanyasem** [ValveSoftware/Proton#1853]
# Whitelist Request
- Name of the game to be whitelisted: Prime World
- Steam AppID of the game: 235340

## System Information
- GPU: GTX 1080
- Driver/LLVM version: nvidia 410.66
- Kernel version: 4.18.16
- Link to full system information report as [Gist](https://gist.github.com/vanyasem/f787469ec8c50cdd3728340ef4c73269)
- Proton version: 3.16-3 Beta

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options)
      are necessary to run the game.
- [x] that no workarounds other than the mentioned ones are necessary.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are
      remaining issues:

**Comment by ghost** [comment]
Good time of day, the game really runs great, the lock works fine, but in battle, after a while, the game crashes, always, here's the log file
[steam-235340.zip](https://github.com/ValveSoftware/Proton/files/4535536/steam-235340.zip)

**Comment by kisak-valve** [comment]
Hello @MitaiGit, does the same thing happen with a mainline Proton build?

**Comment by ghost** [comment]
> Здравствуйте @MitaiGit , происходит ли то же самое с основной сборкой Proton?

Yes, still, this build was the last step, before it were used versions of proton from steam
do I need to attach a system configuration?
